src/axolotl/utils/models.py

```diff
@@ -196,6 +188,13 @@ def load_model(
             )
             load_in_8bit = False
         elif cfg.is_llama_derived_model and "LlamaForCausalLM" in globals():
```

Comment:
will need to edit this line too since `LlamaForCausalLM` isn't in globals anymore